src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/RemoveResourceDesignerStep.cs

```diff
@@ -93,10 +93,6 @@ internal override bool ProcessAssemblyDesigner (AssemblyDefinition assembly)
 		{
 			if (mainDesigner == null)
 				return false;
-			var fileName = assembly.Name.Name + ".dll";
-			if (MonoAndroidHelper.IsFrameworkAssembly (fileName))
```

Comment:
Should this not be using your new `MonoAndroidHelper.IsFromAKnownRuntimePack` ?
The framework assemblies never used the resource designer of old, so we can safely skip those.

Reply:
`IsFromAKnownRuntimePack` only works on MSBuild `ITaskItem`.

I'm thinking all the calls from a linker step, we can remove? They were wrong anyway...

Reply:
Does this still work in Debug mode? We call this during a debug build via the LinkAssembliesNoShrink call.